service/debugger/debugger.go

```diff
@@ -436,6 +436,27 @@ func (d *Debugger) Functions(filter string) ([]string, error) {
 	return regexFilterFuncs(filter, d.process.Funcs())
 }
 
+func (d *Debugger) Types(filter string) ([]string, error) {
```

Comment:
Note: This will have to grab the mutex once https://github.com/derekparker/delve/pull/421 lands.